.vsts-ci.yml

```diff
@@ -8,12 +8,17 @@ trigger:
     - internal/release/3.*
     - internal/release/5.*
     - internal/release/6.*
+    - internal/release/7.*
 
 variables:
 - name: _PublishUsingPipelines
   value: false
-- name: PostBuildSign
-  value: true
+- ${{ if or(startswith(variables['Build.SourceBranch'], 'refs/heads/release/'), startswith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), eq(variables['Build.Reason'], 'Manual')) }}:
```

Comment:
You want manual builds to in build sign?  I assume this is in case we need one off builds or testing (and there's no harm in signing those), correct?

Reply:
Does this need to exclude the internal refs that Daniel mentioned on the SDK version of the change?

Reply:
Nope. That was a typo.